runtime-testsuite/test/org/antlr/v4/test/runtime/descriptors/ParserErrorsDescriptors.java

```diff
@@ -135,7 +135,7 @@
 	public static class InvalidEmptyInput extends BaseParserTestDescriptor {
 		public String input = "";
 		public String output = null;
-		public String errors = "line 1:0 missing ID at '<EOF>'\n";
+		public String errors = "line 1:0 mismatched input '<EOF>' expecting ID\n";
```

Comment:
@parrt This behavior change was caused by removing the `la == TokenConstants.EOF` check in `DefaultErrorStrategy.sync`. Do you have a preference regarding the handling?